stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/io/NextTextRecord.java

```diff
@@ -49,24 +59,77 @@
      */
     private static final int TXT_RECORD_CHANNEL_INDEX = 0;
 
+    /**
+     * Will be the I/O event handler.
+     */
+    private EventManager eventManager = EventManager.getInstance();
+
+    private static final Logger log = LoggerFactory.getLogger(NextTextRecord.class);
+
+    /*
+     * Response obtained after reading record.
+     *
+     * @param result the result obtained after processing the record.
+     * @return the response obtained after reading record.
+     */
+/*
+    private static EventResult readRecordResponse(EventResult<String[], EventContext> result) {
+        BStruct errorStruct;
+        EventContext eventContext = result.getContext();
+        Context context = eventContext.getContext();
+        Throwable error = eventContext.getError();
+        if (null != error) {
+            errorStruct = IOUtils.createError(context, error.getMessage());
+        }
+        String[] fields = result.getResponse();
+        return result;
+    }
```

Comment:
Any reason for commenting this block. Shall we remove this

Reply:
Yes. This is commented since async functions are not available yet. Created the function as a proactive measure so that when the async functions are available this could be uncommented

Reply:
As discussed with @anupama-pathirage added TODOs for these with 3b58422f6c457c12c1ed16661b17ae49a52a17a7  (through i am not a fan of having TODOs) :)